### PR TITLE
(♺Auth): Changed Advert to store a Domain model instead of a String city

### LINKED ISSUE #333: Refactor Backend

### PR DESCRIPTION
**Pull request by lastrucci01**
partitioned the actions/user folder into sub folders, it was getting a bit messy
changed the way Adverts store their location from a string to a domain model

I see amplify changes came through from all we've been doing so pls verify they are correct and I didn't accidentally delete anything

closes #333 #334 #332

### CODE REVIEW

File: libs/tradesman/lib/widgets/notification_card_widget.dart [hunk 1]
```diff
@@ -6,82 +7,79 @@ import 'package:redux_comp/app_state.dart';
...
-                          titleText == "Accepted!"
-                              ? "Your bid was accepted. A chat has been created between you and your client."
-                              : titleText == "Submitted!"
-                                  ? "Your bid was submitted."
-                                  : "A bid was made on your advert.",
-                          style: const TextStyle(
-                              fontSize: 20, color: Colors.white),
-                          maxLines: 3,
-                          overflow: TextOverflow.ellipsis,
+    return StoreProvider<AppState>(
```
Comment (lastrucci01) [inline libs/tradesman/lib/widgets/notification_card_widget.dart:25]: So I think I removed logout action from the view model here cause it didn't make much sense but then I had merge conflicts with develop and I am afraid I may have overwrote your file @caaaramia

Comment (lastrucci01) [reply libs/tradesman/lib/widgets/notification_card_widget.dart:25]: scratch that I removed logout action from tradesman listings, I have no idea what happened here, I think its from develop and it's fine...

Comment (caaaramia) [reply libs/tradesman/lib/widgets/notification_card_widget.dart:25]: all appears to be fine

**Review by Alexander-Muendesi — APPROVED** [review]

**Comment by Alexander-Muendesi** [comment]
From my side the amplify changes seem to be okay

**Pull request merged**